### DISCUSSION TITLE
VPS Experiment Help Needed

### DISCUSSION THREAD

**Discussion by ArjaanAuinger** [apohl79/audiogridder#1123] [Help]
Hi, I am trying to set up audiogridder on a Windows VPS I rented, as a little experiment. I managed to get ZeroTier working as my VPN, but AudioGridder doesn't see my server. Any Ideas? Is this even possible? If not, why? I can't find any infos specifically about networking and haven't had enough time to work through the code myself. Any tips welcome :)

**Comment by oldtos** [comment]
I found this.. I expect latency... I wish more detail had been given.... but read here
`https://github.com/apohl79/audiogridder/discussions/670`

**Comment by ArjaanAuinger** [comment]
Thank you very much, I did not find that, although using the search function! So Wusik managed to get it working with ZeroTier, now I am curious how, because our setup seems the same, but I can't connect to my server. And apohl wrote, he doesn't see any problem to connect to a remote server. Actually, how? It automatically finds servers in your local network, can I just press "add" in the little server dropdown and add any ip-address, even if its not in the local network?

**Comment by oldtos** [comment]
I think I would get wireshark .. ( they make a portable no install version)  and watch AG plugin .. see port it needs.. and make sure firewalls and such allow AG,,,  while ZeroTier can get through .. does not mean Audiogridder can....
